azerothcore/azerothcore-wotlk
# fix(Core/Spells): Implemented PROC_EX_ONLY_FIRST_TICK.

Fixes #7677



## Issues Addressed:

- Closes #7677
- Closes https://github.com/chromiecraft/chromiecraft/issues/1502

## SOURCE:


## Tests Performed:

- Not tested

## How to Test the Changes:

1. Be a mage.
2. Get `Arcane Concentration` talent (5 points)
3. Cast Blizzard/Arcane Missiles 
4. See if Clearcasting proc rate is bigger than 10%.
More info [here](#7677)

Omen of Clarity (druid) and Elemental Focus (Shaman) should be tested as well. 
These spells should be tested with some AOE spells (Hurricane/Tranquility for druids / Chain Heal/Chain Lighting for shamans) as well as some channeled or single spells to ensure there are no new bugs.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
